src/fwup_apply.c

```diff
@@ -416,7 +416,7 @@ int fwup_apply(const char *fw_filename, const char *task_prefix, int output_fd,
     OK_OR_CLEANUP(fatfs_ptr_callback(&fctx, -1, NULL));
 
     // Flush a subarchive that's being built.
-    OK_OR_CLEANUP(subarchive_ptr_callback(&fctx, "", NULL, NULL));
+    OK_OR_CLEANUP(subarchive_ptr_callback(&fctx, NULL, NULL, NULL));
```

Comment:
I fixed this in the `windows` branch. If you fetch and rebase, the `subarchive_ptr_callback` commit should disappear. Great catch!